infra/ansible/publish_torchprime_e2e_test_docker.sh

```diff
@@ -0,0 +1,63 @@
+#!/bin/bash
+
+# This script builds and pushes a docker image to be used for torchprime E2E tests.
+#
+# torchprime is a reference implementation of models using PyTorch/XLA:
+# https://github.com/AI-Hypercomputer/torchprime.
+#
+# The purpose of building a docker image here is to trigger torchprime E2E tests
+# from PyTorch/XLA PRs and post-submits. The reason for running torchprime tests
+# on PyTorch/XLA changes is to ensure that torchprime models are not broken.
+# See https://github.com/AI-Hypercomputer/torchprime/issues/161 for the detailed
+# motivation.
+
+set -ex
+
+# Check required environment variables
+if [ -z "${DEFAULT_CONTEXT_PATH}" ]; then
+  echo "ERROR: DEFAULT_CONTEXT_PATH is not set"
+  exit 1
+fi
+if [ -z "${DOCKER_IMAGE_NAME}" ]; then
+  echo "ERROR: DOCKER_IMAGE_NAME is not set"
+  exit 1
+fi
+if [ -z "${DOCKER_IMAGE_TAG}" ]; then
+  echo "ERROR: DOCKER_IMAGE_TAG is not set"
+  exit 1
+fi
+if [ -z "${DOCKER_PROJECT}" ]; then
+  echo "ERROR: DOCKER_PROJECT is not set"
+  exit 1
+fi
+
+export IMAGE_NAME="gcr.io/${DOCKER_PROJECT}/${DOCKER_IMAGE_NAME}:${DOCKER_IMAGE_TAG}"
```

Comment:
This name can be confused with `DOCKER_IMAGE_NAME`. `IMAGE_PATH` or `IMAGE_URL`?